src/app/submission/sections/form/section-form.component.ts

```diff
@@ -144,16 +154,20 @@ export class SubmissionSectionformComponent extends SectionModelComponent {
   onSectionInit() {
     this.pathCombiner = new JsonPatchOperationPathCombiner('sections', this.sectionData.id);
     this.formId = this.formService.getUniqueId(this.sectionData.id);
-
     this.formConfigService.getConfigByHref(this.sectionData.config).pipe(
       map((configData: ConfigData) => configData.payload),
       tap((config: SubmissionFormsModel) => this.formConfig = config),
-      flatMap(() => this.sectionService.getSectionData(this.submissionId, this.sectionData.id)),
+      flatMap(() =>
+        combineLatestObservable(
+          this.sectionService.getSectionData(this.submissionId, this.sectionData.id),
+          this.workspaceItemDataService.findById(this.submissionId).pipe(getSucceededRemoteData(), map((wsiRD: RemoteData<WorkspaceItem>) => wsiRD.payload))
+        )),
       take(1))
-      .subscribe((sectionData: WorkspaceitemSectionFormObject) => {
+      .subscribe(([sectionData, workspaceItem]: [WorkspaceitemSectionFormObject, WorkspaceItem]) => {
         if (isUndefined(this.formModel)) {
           this.sectionData.errors = [];
           // Is the first loading so init form
+          this.workspaceItem = workspaceItem;
```

Comment:
Tiny thing, but you put this new line right after a comment that refers to the *next* line (the initForm call).  So, the comment above should be moved below.

Reply:
Moved the comment to the right line.